### ISSUE TITLE
PluginInstall ignores all plugins

### ISSUE THREAD

**Issue by Nimdis** [gmarik/Vundle.vim#611]
Hello I've just installed Ubuntu 15.04 and tried to install and configure vim.
I've installed vim (7.4.488) from repository, cloned this repository etc.
My current vimrc looks so

```
set nocompatible
filetype off

set rtp+=~/.vim/bundle/Vundle.vim/

call vundle#begin()

Plugin 'gmarik/Vundle.vim'

Plugin 'bling/vim-airline'
Plugin 'garbas/vim-snipmate'

call vundle#end()

filetype plugin indent on
```

I run vim and execute ```:PluginInstall``` but it's only installing vundle itself.

```
. Plugin 'gmarik/Vundle.vim' 
* Helptags
```

What am I doing wrong?

**Comment by z3t0** [comment]
Are you sure you cloned Vundle to the correct directory?

**Comment by z3t0** [comment]
Is it possible that you are editing the wrong vimrc?

**Comment by Nimdis** [comment]
@z3t0 
Yes, I'm sure. I've tried many time to edit config and change path of vundle directory, removed and cloned many times, but nothing helped. That's weird because it's happening only on ubuntu. On Mac OS X everything works well.
I've the following line in my ```.vimrc```

```
source ~/.vim/vimrc
```

Content of ```~/.vim/vimrc``` you can see in my massage above.

**Comment by z3t0** [comment]
Hmm I thought the vimrc should be in ~/.vimrc

**Comment by z3t0** [comment]
@Nimdis I'll be on the gitter for Vundle

**Comment by Nimdis** [comment]
output of the ```:version``` command.
```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Dec  9 2014 17:36:18)
Included patches: 1-488
Modified by pkg-vim-maintainers@lists.alioth.debian.org
Compiled by buildd@
Huge version without GUI.  Features included (+) or not (-):
+acl             +cmdline_compl   +diff            +find_in_path    +keymap          +modify_fname    +mouse_xterm     +profile         -sniff           +termresponse    +vreplace        -xterm_save
+arabic          +cmdline_hist    +digraphs        +float           +langmap         +mouse           +multi_byte      +python          +startuptime     +textobjects     +wildignore      -xpm
+autocmd         +cmdline_info    -dnd             +folding         +libcall         -mouseshape      +multi_lang      -python3         +statusline      +title           +wildmenu
-balloon_eval    +comments        -ebcdic          -footer          +linebreak       +mouse_dec       -mzscheme        +quickfix        -sun_workshop    -toolbar         +windows
-browse          +conceal         +emacs_tags      +fork()          +lispindent      +mouse_gpm       +netbeans_intg   +reltime         +syntax          +user_commands   +writebackup
++builtin_terms  +cryptv          +eval            +gettext         +listcmds        -mouse_jsbterm   +path_extra      +rightleft       +tag_binary      +vertsplit       -X11
+byte_offset     +cscope          +ex_extra        -hangul_input    +localmap        +mouse_netterm   -perl            -ruby            +tag_old_static  +virtualedit     -xfontset
+cindent         +cursorbind      +extra_search    +iconv           -lua             +mouse_sgr       +persistent_undo +scrollbind      -tag_any_white   +visual          -xim
-clientserver    +cursorshape     +farsi           +insert_expand   +menu            -mouse_sysmouse  +postscript      +signs           -tcl             +visualextra     -xsmp
-clipboard       +dialog_con      +file_in_path    +jumplist        +mksession       +mouse_urxvt     +printer         +smartindent     +terminfo        +viminfo         -xterm_clipboard
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -g -O2 -fPIE -fstack-protector-strong -Wformat -Werror=format-security -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: gcc   -Wl,-Bsymbolic-functions -fPIE -pie -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -o vim        -lm -ltinfo -lnsl  -lselinux  -lacl -lattr -lgpm -ldl    -L/usr/lib/python2.7/config-x86_64-linux-g
nu -lpython2.7 -lpthread -ldl -lutil -lm -Xlinker -export-dynamic -Wl,-O1 -Wl,-Bsymbolic-functions
```

**Comment by afei** [comment]
Hi there, I have same situation in CentOS, and the version of VIM is 7.4.170. 
And my .vimrc file was in ~/.vimrc.

any update? @z3t0 @Nimdis

**Comment by z3t0** [comment]
Please share the output of :version

On Wednesday, 27 April 2016, Alfred <notifications@github.com> wrote:

> Hi there, I have same situation in CentOS, and the version of VIM is
> 7.4.170.
> And my .vimrc file was in ~/.vimrc.
>
> any update? @z3t0 <https://github.com/z3t0> @Nimdis
> <https://github.com/Nimdis>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/VundleVim/Vundle.vim/issues/611#issuecomment-215313880>
>


-- 
Thanks,
Rafi Khan

**Comment by afei** [comment]
Thank you for your help. Please find the output below:
[vim_version.txt](https://github.com/VundleVim/Vundle.vim/files/240377/vim_version.txt)

**Comment by afei** [comment]
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Jun 10 2014 06:55:55)
Included patches: 1-160
Modified by <bugzilla@redhat.com>
Compiled by <bugzilla@redhat.com>
Huge version without GUI.  Features included (+) or not (-):
+acl             +conceal         +farsi           +libcall         +mouse_netterm   +profile         +syntax          +visualextra
+arabic          +cryptv          +file_in_path    +linebreak       +mouse_sgr       +python/dyn      +tag_binary      +viminfo
+autocmd         +cscope          +find_in_path    +lispindent      -mouse_sysmouse  -python3         +tag_old_static  +vreplace
-balloon_eval    +cursorbind      +float           +listcmds        +mouse_urxvt     +quickfix        -tag_any_white   +wildignore
-browse          +cursorshape     +folding         +localmap        +mouse_xterm     +reltime         -tcl             +wildmenu
++builtin_terms  +dialog_con      -footer          -lua             +multi_byte      +rightleft       +terminfo        +windows
+byte_offset     +diff            +fork()          +menu            +multi_lang      +ruby/dyn        +termresponse    +writebackup
+cindent         +digraphs        +gettext         +mksession       -mzscheme        +scrollbind      +textobjects     -X11
-clientserver    -dnd             -hangul_input    +modify_fname    +netbeans_intg   +signs           +title           -xfontset
-clipboard       -ebcdic          +iconv           +mouse           +path_extra      +smartindent     -toolbar         -xim
+cmdline_compl   +emacs_tags      +insert_expand   -mouseshape      +perl            -sniff           +user_commands   -xsmp
+cmdline_hist    +eval            +jumplist        +mouse_dec       +persistent_undo +startuptime     +vertsplit       -xterm_clipboard
+cmdline_info    +ex_extra        +keymap          +mouse_gpm       +postscript      +statusline      +virtualedit     -xterm_save
+comments        +extra_search    +langmap         -mouse_jsbterm   +printer         -sun_workshop    +visual          -xpm
   system vimrc file: "/etc/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/etc"
 f-b for $VIMRUNTIME: "/usr/share/vim/vim74"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -O2 -g -pipe -Wall -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1      
Linking: gcc   -L. -Wl,-z,relro -fstack-protector -rdynamic -Wl,-export-dynamic -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib64/perl5/CORE  -Wl,-z,relro  -L/usr/local/lib -Wl,--as-needed -o vim        -lm -lnsl  -lselinux  -lncurses -lacl -lattr -lgpm -ldl   -Wl,--enable-new-dtags -Wl,-rpath,/usr/lib64/perl5/CORE  -fstack-protector  -L/usr/lib64/perl5/CORE -lperl -lresolv -lnsl -ldl -lm -lcrypt -lutil -lpthread -lc

**Comment by nramirezuy** [comment]
I just ran into this issue. Got it fixed by destroying everything (removing Vundle.vim) and re-cloning the repo.
